src/mudlet-lua/lua/LuaGlobal.lua

```diff
@@ -146,25 +154,61 @@ if getMudletLuaDefaultPaths then
   end
 end
 
+if package.config:sub(1,1) == '\\' then
```

Comment:
We have a `getOS()` function that should be used for this!

Reply:
Humm, I will need to _improve_ that for other *nixes - probably with a system call to `uname` - anyhow the test I am using is _exactly_ what is needed in the context it is being used - given what the four values stored in the `package.config` string seem to be for...

Reply:
You could just do
```lua
for i, path in ipairs(prefixes) do
  prefixes[i] = string.gsub(path, '[/\\]', package.config:sub(1,1))
  if debugLoading then
    echo([[Directory separator conversion: "]] .. path .. [[" becomes: "]] .. prefixes[i] .. [["
]])
  end
end
```
And get rid of the `if`s. I hear there are also OSes that have other separators.

Reply:
> You could just do

:+1: Yeah that does seem a little more elegant.

> I hear there are also OSes that have other separators.

Ohhh, you think there may be issues if someone wants to port to Raspberry Pis running [RISC OS Open](https://en.wikipedia.org/wiki/RISC_OS_Open) - which seems to be the most likely [_other_](https://en.wikipedia.org/wiki/Path_(computing)#Representations_of_paths_by_operating_system_and_shell) directory separator usage... :rofl: